### PR TITLE
Update Dockerfile to Ubuntu 20.04

### PR DESCRIPTION
**Pull request by qhdwight**

### CODE REVIEW

**Comment by cgiger00** [comment]
changing this to a different branch for now?

**Comment by qhdwight** [comment]
> changing this to a different branch for now?

Sure we should probably make sure it passes travis, but I'm not exactly sure how to do that

**Comment by qhdwight** [comment]
Nevermind I see that it does
`docker run --rm -it -v $(pwd):/root -v /dev/tty:/dev/ttyUSB0 -v /dev/null:/run/udev:ro -v /dev/tty:/dev/i2c-0 -v /dev/tty:/dev/i2c-1 -v /dev/tty:/dev/i2c-2 umrover1/travis:latest jarvis build -a .`
So I can try that with the new image.

**Comment by cgiger00** [comment]
yeah exactly

**Comment by cgiger00** [comment]
it's not going to pass, the new linter makes some of the gps & lcm bridge server code fail, in https://github.com/umrover/mrover-workspace/pull/914/files i just added #noqa to the lines that don't pass for now

**Comment by cgiger00** [comment]
nice

File: rundocker.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+xhost local:root && \
+docker run -it --env="DISPLAY" \
+  --volume=${HOME}:/mnt/ \
+  --volume="/tmp/.X11-unix:/tmp/.X11-unix:rw" \
+  --gpus all -e NVIDIA_DRIVER_CAPABILITIES=all --privileged --net=host \
+  umrover1/travis:latest
```
Comment (cgiger00) [inline rundocker.sh:6]: ugh i realized post merging, should this be travis:20.04?

Comment (qhdwight) [reply rundocker.sh:6]: probaby, all though I changed this script to use $1 instead (so you can do `./rundocker <image name>`)

**Comment by niwhsa9** [comment]
did OpenCV 3.2 actually compile and link correctly with this?

**Pull request merged**